CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.0-rc.17 (2017-08-23)
+
+- Created `--serve`/`-s` flag for `skyux build` to easily allow a built SPA to be accessed.  Can be used in conjunction with the `--launch` and `--browser` flags. [#219](https://github.com/blackbaud/skyux-builder/pull/219)
```

Comment:
I'd add "the" after "Created."

I'd change "to easily allow a built SPA to be accessed" to "to serve the SPA locally when it builds."

I'd "Can be used" to "Can use."